.env.example

```
@@ -1,37 +1,18 @@
+DB_USERNAME=
+DB_PASSWORD=
+
+JWT_SECRET=
+ADMIN_JWT_SECRET=
+API_TOKEN_SALT=
+TRANSFER_TOKEN_SALT=
+ENCRYPTION_KEY=
+
+CLOUDINARY_NAME=
+CLOUDINARY_KEY=
+CLOUDINARY_SECRET=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)